static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -50,16 +41,43 @@ const initialise = (window) => {
           }
         : { syncEnabled: false };
 
+        let consentManagement = {}
+        if (config.get('switches.consentManagement', false)) {
+			getLocale().then((country) => {
+				switch (country) {
+					case 'AU':
+					case 'US':
+						// https://docs.prebid.org/dev-docs/modules/consentManagementUsp.html
+						consentManagement = {
+							usp: {
+								cmpApi: 'iab',
+								timeout: 1500,
+							},
+						};
+						break;
+					case 'GB':
+					default:
+						// https://docs.prebid.org/dev-docs/modules/consentManagement.html
+						consentManagement = {
+							gdpr: {
+								cmpApi: 'iab',
+								timeout: 200,
+								defaultGdprScope: true,
+							},
+						};
+						break;
+				}
+			});
+		}
+
```

Comment:
Make the creation of `consentManagement` a function? Then we don't need the `let` ?

Reply:
Yes, actually we should not set the value of `consentManagement` at all if we're not enabling a CMP, so I'll move the conditional further down.